README.md

```diff
@@ -61,6 +61,13 @@ cargo watch -s "cargo check"
 
 See [instructions](proto/README.md) in the proto subdirectory.
 
+Both rust-protobuf and prost are supported. rust-protobuf is used by default. If you wish to use prost, you need to enable the prost features.
+
+```
+[dependencies]
+raft = { git = "https://github.com/pingcap/raft-rs.git", default-features = false, features = ["repr-prost"] }
+```
```

Comment:
> To make a project support both rust-protobuf and prost at the same time, you just need to add two features repr-protobuf and repr-prost. And then use jinkela-build to generate codes and include the generated sources in module files.

What about this part?

Reply:
It's instructions for using the jinkela library itself, not related to raft.

Reply:
Can you add it to https://github.com/BusyJay/jinkela/blob/master/README.md ?

Reply:
Of course. May do it when I wake up.